release_notes/369-release-notes.adoc

```diff
@@ -12,59 +12,73 @@ toc::[]
 [id="new-features"]
 == New features
 
+[id="openshift-container-registry]
+=== Enhanced support for OpenShift Container Registry (OCR)
+With {product-title} 3.69, a lightweight version of scanner is delivered as part of secured cluster services on OpenShift by default to more effectively scan OpenShift Container Registry. For non-OpenShift environments this scanner is optional. OpenShift users not using the {product-title} are advised to update their helm charts to take advantage of these new capabilites.
```

Comment:
I guess it is optional, but we should also make it clear it's not going to add them any value. It's really only for OpenShift users at this time